src/pages/resources/faq.md

```diff
@@ -57,6 +57,16 @@ Yes, you can safely upgrade extensions within the mobile property. All changes a
 
 Additionally, the installation instructions in the mobile property will be refreshed to include the latest versions of mobile platform extensions. These instructions are primarily necessary during the development phase and will be updated even before the property is published.
 
+## How often should I upgrade extensions in the Data Collection UI for published mobile apps?
+
+It is recommended to always update to the latest version to access new features and show installation instructions for the latest major releases. You can upgrade extensions within the mobile property safely, as updates are backward compatible and won't affect apps already using the published configuration.
```

Comment:
I don't love using "It is recommended..." as part of the languaging.

I'd go with "You should always update..."